dotnet/samples/NCalcSkills/LanguageCalculatorSkill.cs

```diff
@@ -75,6 +75,8 @@ public LanguageCalculatorSkill(IKernel kernel)
             maxTokens: 256,
             temperature: 0.0,
             topP: 1);
+
+        this._kernel = kernel;
```

Comment:
In this scenario, we really don't want to expose these 'inner' semantic functions to the outer kernel nor use them on invocation below. Do we have work tracked for making it easier to create a kernel with the available AIServices?

Reply:
you are right, I did it wrong here.